src/views/Layers.vue

```diff
@@ -1,36 +1,62 @@
 <template>
   <v-card class="layers" flat>
-    <v-card-text>
-      Lorem ipsum dolor sit amet.
-    </v-card-text>
+    <v-select
+      v-model="selectedTags"
+      class="px-4 pt-4 pb-1"
+      label="Filter by tag"
+      multiple
+      dense
+      outlined
+      hide-details
+      :items="layerTags"
+    />
     <layer-list-controls
-      v-if="layers"
-      :layers="layers"
+      v-if="layersForTree"
+      :layers="layersForTree"
       @active-layers-change="onActiveLayerChange"
       @layer-sorting-change="onLayerSortingChange"
     />
   </v-card>
 </template>
 
 <script>
-  import { mapActions, mapState } from 'vuex'
+  import { mapActions, mapGetters } from 'vuex'
```

Comment:
I never ever use state directly in templates (you can accidentally mutate state, never a getter)